### PR TITLE
@alloy => Configurable threshold for sending users to the new artist page

### PR DESCRIPTION
**Pull request by mzikherman**
When we're ready, this allows us to set a configurable number in the ENV, `PERCENT_NEW_ARTIST_PAGE=2` (which means 20% of users will get the new artist page).

This uses being cookied with a number from 0-9, which makes the transition smoother for users. You'll keep seeing the old page until your in a bucket that gets the new one, and from then on you'll always see the new one.

### CODE REVIEW

File: src/desktop/apps/artist/maybe_new_artist_page.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// TODO: Read from ENV.
+const NEW_ARTIST_PAGE_THRESHOLD = 3
+
+module.exports = (_req, res, next) => {
+  console.log(res.locals.sd.NEW_ARTIST_PAGE)
+  if (parseInt(res.locals.sd.NEW_ARTIST_PAGE, 10) < NEW_ARTIST_PAGE_THRESHOLD) {
+    return next('route')
```
Comment (mzikherman) [inline src/desktop/apps/artist/maybe_new_artist_page.ts]: This nexts out of the artist app, and so you'll get artist2 (served under /artist)

File: src/desktop/apps/artist/maybe_new_artist_page.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  } else {
+    return next()
```
Comment (mzikherman) [inline src/desktop/apps/artist/maybe_new_artist_page.ts]: This leaves you in the existing artist page.

File: src/desktop/apps/artist/maybe_new_artist_page.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// TODO: Read from ENV.
+const NEW_ARTIST_PAGE_THRESHOLD = 3
+
+module.exports = (_req, res, next) => {
+  console.log(res.locals.sd.NEW_ARTIST_PAGE)
```
Comment (alloy) [inline src/desktop/apps/artist/maybe_new_artist_page.ts]: Stray loggy?

Comment (mzikherman) [reply src/desktop/apps/artist/maybe_new_artist_page.ts]: yes, was just testing! Still need to switch to grab the const from ENV/config, will remove the log too.

**Comment by mzikherman** [comment]
Will reopen from a new branch, Circle CI is having issues.

**Pull request closed without merging**